### PR TITLE
fix(115): Add response schemas for versions, contexts, stats, and status

### PR DESCRIPTION
**Pull request by Filbird**
## Context

There are response object schemas missing for several endpoints as detailed in screwdriver-cd/screwdriver#115

Pending merging of https://github.com/screwdriver-cd/data-schema/pull/222

## Objective

- [x] Add response object schemas for:
  - [x] versions
  - [x] contexts
  - [x] stats
  - [x] status 
- [x] Remove incorrect response schema for step update
- [x] Add auth tag to login and logout routes

## References

Swagger documentation:
https://api.screwdriver.cd/v4/documentation

### CODE REVIEW

File: plugins/builds/steps/update.js [hunk 1]
```diff
@@ -57,9 +57,6 @@ module.exports = () => ({
                 .then(build => reply(build.steps[stepIndex]).code(200))
                 .catch(err => reply(boom.wrap(err)));
         },
-        response: {
```
Comment (FenrirUnbound) [inline plugins/builds/steps/update.js]: I might be missing context on this, but why was this particular schema dropped?

Comment (Filbird) [reply plugins/builds/steps/update.js]: I may be wrong but it seems like this response schema was intended only for GETs on this endpoint.

Comment (tkyi) [reply plugins/builds/steps/update.js]: I think the schema still applies to the PUT endpoint; they were probably just reusing it

Comment (Filbird) [reply plugins/builds/steps/update.js]: ooh, my mistake, I will remove that commit!

**Pull request merged**